setup/security/authentication/ldap/index.md

```diff
@@ -31,9 +31,11 @@ successful, the bind is successful and a session is established.
 
 Use `hal config` to enable and configure LDAP. Here's an example:
 
+`hal config security authn ldap edit --user-dn-pattern="uid={0},uid=users" --url=ldaps://ldap.my-organization.com:10636/dc=my-organization,dc=com`
+
 `hal config security authn ldap enable`
 
-`hal config security authn ldap edit --user-dn-pattern="uid={0},uid=users" --url=ldaps://ldap.my-organization.com:10636/dc=my-organization,dc=com`
+You should enable after configured LDAP.
```

Comment:
@hareku 
What about omitting this sentence. That is, you've swapped the order of the two commands, maybe it's not necessary to describe why?

Reply:
@dorbin That is true! I removed it.